modules/ROOT/pages/govern-api-instances.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 
 
-API Manager is closely integrated with xref:api-governance::index.adoc[Anypoint API Governance] to govern aspects of API instances with governance rulesets. Applying a governance ruleset to your API ensures that every instance of that API conforms to standards such as what policies are applied and if a TLS context is required.  
+API Manager is closely integrated with xref:api-governance::index.adoc[Anypoint API Governance]. Use xref:api-governance::index.adoc[Anypoint API Governance] to govern API instances with governance rulesets. Applying a governance ruleset to your API ensures that every instance of that API conforms to standards, such as which policies are applied, or whether a TLS context is required.
```

Comment:
```suggestion
API Manager is closely integrated with xref:api-governance::index.adoc[Anypoint API Governance]. API Governance helps you ensure that every instance of a governed API conforms to standards, such as which policies are applied or whether a TLS context is required.
```

Reply:
- I'm not sure it's necessary to say the first part about being closely integrated. Is that important to state? I know I reviewed this before and I don't want to nitpick too much if you want to include that.
- The current edit has the link to API Gov doc twice in back to back sentences, which is not necessary. 
- It's a bit much to start trying to explain rulesets in the API Manager doc, as there is a lot more to configuring API Governance than just applying a ruleset. I suggest bringing it up a level, as those who want more will click the link to the API Governance doc.